app/src/main/java/ru/practicum/android/diploma/data/network/CountriesRequest.kt

```diff
@@ -0,0 +1,3 @@
+package ru.practicum.android.diploma.data.network
+
+class CountriesRequest
```

Comment:
нужно добавить текст запроса
```suggestion
data class CountriesRequest(val expression: String)
```